src/endpoint/s3/s3_utils.js

```diff
@@ -322,6 +324,18 @@ function set_response_object_md(res, object_md) {
     }
 }
 
+/**
+ * @param {nb.S3Response} res 
+ * @param {Array<string>} [supported_storage_classes]
+ */
+function set_response_supported_storage_classes(res, supported_storage_classes) {
+    if (!supported_storage_classes) {
+        supported_storage_classes = [STORAGE_CLASS_STANDARD];
+    }
+
+    res.setHeader(X_NOOBAA_AVAILABLE_STORAGE_CLASSES, supported_storage_classes);
```

Comment:
Will this header be sent if the configuration is such that we get an empty array? 
Lets just make sure we behave in a determined way in case so users will not be confused.

Reply:
The headers aren't sent in that case I believe (at least I couldn't see them in the outputs)

PS: I have added screenshots in the PR description.